tests/e2e/e2e.sh

```diff
@@ -75,25 +31,36 @@ ISTIO_DIR="${GOPATH}/src/istio.io/istio"
 # Create a clone of the Istio repository
 if [[ ! -d "${ISTIO_DIR}" ]]
 then
-  git clone https://github.com/sdake/istio.git "${ISTIO_DIR}"
+  git clone https://github.com/istio/istio.git "${ISTIO_DIR}"
 fi
 
 # Create an operator manifest from the default control plane configuration
-
-operator_manifest_files=( "deploy/namespace.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_crd.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml" "deploy/service_account.yaml" "deploy/clusterrole.yaml" "deploy/clusterrole_binding.yaml" "deploy/service.yaml" "deploy/operator.yaml" )
+cd "${GOPATH}/src/istio.io/operator"
+operator_manifest_files=( "deploy/namespace.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_crd.yaml" "deploy/service_account.yaml" "deploy/clusterrole.yaml" "deploy/clusterrole_binding.yaml" "deploy/service.yaml" "deploy/operator.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml" )
```

Comment:
the ordering shouldn't matter - not sure this change is correct.

Reply:
It will fails if we deploy the icp cr a bit ahead. It complains that no crd found. By making cr deploy the last one, we can make sure no such problem happens.

Reply:
thats bad news - but not unexpected.  Thanks.